tests/plugins/wysiwygarea/manual/detachobserver.md

```diff
@@ -0,0 +1,11 @@
+@bender-tags: 4.17.2, feature, 4462
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea,toolbar,basicstyles,link,format,sourcearea,elementspath,undo
+
+1. Open dev console to track possible errors.
```

Comment:
```suggestion
1. Open the dev console to track possible errors.
```
Or maybe
```suggestion
1. Open a browser console to track possible errors.
```

Reply:
Seems that we are using the `dev console` quite often - however I'll change it.